src/app/legacy/containers/PageHandlers/withOptimizelyProvider/index.tsx

```diff
@@ -72,34 +78,49 @@ const getReferrer = () => {
   return null;
 };
 
-const withOptimizelyProvider = Component => {
-  return props => {
-    const { service } = use(ServiceContext);
-    const isStoryBook = process.env.STORYBOOK;
-    const disableOptimizely = isStoryBook || isInCypress;
-
-    if (disableOptimizely) return <Component {...props} />;
+export const COUNTRY_CODES_TO_EXPERIMENT = [
```

Comment:
These seem to all conform to ISO 3166-1 alpha-2:

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2